chart/kubeapps/templates/_helpers.tpl

```diff
@@ -102,10 +112,10 @@ Create name for the apprepository cleanup job
 {{- end -}}
 
 {{/*
-Create name for the mongodb secret bootstrap job
+Create name for the db-secret secret bootstrap job
 */}}
-{{- define "kubeapps.mongodb-jobs-cleanup.fullname" -}}
-{{ template "kubeapps.fullname" . }}-internal-mongodb-jobs-cleanup
+{{- define "kubeapps.db-secret-jobs-cleanup.fullname" -}}
+{{ template "kubeapps.fullname" . }}-internal-db-secret-jobs-cleanup
```

Comment:
Will this break upgrades of existing Kubeapps? (if the k8s secret name changes, I mean).

Reply:
In theory it should not (it will only break if someone tries to upgrade from mongo to postgres) but I will double check that before merging/releasing this feature.